src/http/status.cr

```diff
@@ -0,0 +1,174 @@
+# An enum that provides additional support around HTTP status codes.
+#
+# Based on [Hypertext Transfer Protocol (HTTP) Status Code Registry](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml)
+#
+# It provides constants for the defined HTTP status codes as well as helper
+# methods to easily identify the type of response.
+enum HTTP::Status
+  CONTINUE                        = 100
```

Comment:
I'd prefer if there were in `PascalCase`.

Reply:
I don't oppose standardization here, but I don't think there is an established rule for either case.

Reply:
You mean CamelCase? I find also this case closer to the actual status code message, they are never in all caps.

Reply:
@RX14 that's a bikeshedding, it can be changed in another PR if needed along with other enums like `Flate::Strategy`